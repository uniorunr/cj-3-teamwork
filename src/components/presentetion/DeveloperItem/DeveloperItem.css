.developer-item {
  text-align: center;
}

.developer-item img {
  width: 50px;
  border-radius: 50%;
}
