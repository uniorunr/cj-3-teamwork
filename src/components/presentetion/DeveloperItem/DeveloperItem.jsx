import React from 'react';
import PropTypes from 'prop-types';
import './DeveloperItem.css';

const DeveloperItem = ({ img, name }) => (
  <div className="developer-item">
    <img src={img} alt={name} />
    <p>{name}</p>
  </div>
);

DeveloperItem.propTypes = {
  img: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired,
};

export default DeveloperItem;
