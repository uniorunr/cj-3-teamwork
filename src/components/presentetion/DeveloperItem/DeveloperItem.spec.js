import React from 'react';
import renderer from 'react-test-renderer';
import DeveloperItem from './DeveloperItem';

it('renders correctly', () => {
  const tree = renderer
    .create(<DeveloperItem img="img" name="dev" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
