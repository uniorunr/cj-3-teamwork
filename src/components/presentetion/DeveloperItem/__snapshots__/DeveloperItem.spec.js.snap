// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="developer-item"
>
  <img
    alt="dev"
    src="img"
  />
  <p>
    dev
  </p>
</div>
`;
